website/src/utils/getRelease.js

```diff
@@ -0,0 +1,10 @@
+import { repository, version as pkgVersion } from "../../../package.json"
+
+const version = `v${pkgVersion}`;
+
+const url = () => `https://github.com/${repository}/releases/tag/${version}`;
```

Comment:
Let's use `https://github.com/${repository}/blob/master/CHANGELOG.md`.

Also, could you run `npm run lint -- --fix`?

Reply:
Oops, sure!